packages/manager/modules/overthebox/src/overthebox/overTheBox.routing.js

```diff
@@ -0,0 +1,39 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('overTheBoxes.overTheBox', {
+    url: '/:serviceName',
+    abstract: true,
+    component: 'ovhManagerOverTheBoxComponent',
+    resolve: {
+      service: /* @ngInject */ (OvhApiOverTheBox, $stateParams) =>
+        OvhApiOverTheBox.v6()
+          .get({ serviceName: $stateParams.serviceName })
+          .$promise.then((service) => service),
+      serviceName: /* @ngInject */ ($transition$) =>
+        $transition$.params().serviceName,
+      $title(translations, $translate, $stateParams, OvhApiOverTheBox) {
```

Comment:
Please use explicit annotation: 
```suggestion
      $title: /* @ngInject */ (translations, $translate, $stateParams, OvhApiOverTheBox) => {
```